corehq/apps/export/forms.py

```diff
@@ -1203,7 +1144,7 @@ class FilterCaseESExportDownloadForm(EmwfFilterExportMixin, GenericFilterCaseExp
     def __init__(self, domain_object, timezone, *args, **kwargs):
         self.timezone = timezone
         self.skip_layout = True
-        super(FilterCaseESExportDownloadForm, self).__init__(domain_object, timezone, *args, **kwargs)
+        super(FilterCaseESExportDownloadForm, self).__init__(domain_object, *args, **kwargs)
```

Comment:
This change mimics the functionality of the now defunct `GenericFilterCaseExportDownloadForm`